packages/flame_lottie/test/flame_lottie_test.dart

```diff
@@ -33,23 +33,17 @@ void main() {
 
       final mockAsset = FakeAssetBundle({'logo.json': logoData});
 
-      LottieComposition? composition;
-
       final asset = Lottie.asset(
         'logo.json',
         bundle: mockAsset,
-        onLoaded: (c) {
-          composition = c;
-        },
       );
 
-      composition = await loadLottie(asset);
```

Comment:
variable is set twice, trips https://dcm.dev/docs/rules/common/avoid-unnecessary-reassignment